src/containers/buttons/GenerateSuiteButton.js

```diff
@@ -0,0 +1,137 @@
+import React, {Component} from "react";
+import {bindActionCreators} from "redux";
+import {connect} from "react-redux";
+import xmlbuilder from "xmlbuilder";
+import {saveAs} from "file-saver";
+
+
+class GenerateSuiteButton extends Component {
+
+  handleSuiteGenerating() {
+    const projectTests = this.props.project[0].tests;
+    const suiteElement = xmlbuilder.create('suite', {encoding: "utf-8"});
+    suiteElement.att('name', this.props.project[0].suite);
```

Comment:
Can you chain that method calls to improve readability?

Reply:
Done.